client/scripts/controllers/chain/hydradx/main.ts

```diff
@@ -0,0 +1,22 @@
+import { ChainClass, NodeInfo } from 'models';
+import { IApp } from 'state';
+import Substrate from '../substrate/main';
+import SpecTypes from './spec';
+
+class HydraDX extends Substrate {
+  constructor(n: NodeInfo, app: IApp) {
+    super(n, app, ChainClass.HydraDX);
+    this.signaling.disable();
```

Comment:
I would check the hydra runtime to ensure that these are the only modules we need to disable.

Reply:
seems resolved